### DISCUSSION TITLE
New release 0.9.0

### DISCUSSION THREAD

**Discussion by patrickpichler** [go-shiori/shiori-web-ext#45] [Feature requests]
Hey @fmartingr,

Would it be possible to get a new release? 

The keep logged in feature added with https://github.com/go-shiori/shiori-web-ext/commit/20967de2f7b03571db72b46fdaa97f34a5f2da40 is pretty useful.

**Comment by fmartingr** [comment]
Hey @patrickpichler, I just released [v0.8.7](https://github.com/go-shiori/shiori-web-ext/releases/tag/v0.8.7) from latest `master`. Let me know how it rolls.

**Reply by patrickpichler** [reply]
Awesome! Thank you 🙂
